cocotb/regression.py

```diff
@@ -115,7 +115,11 @@ def __init__(self, dut: SimHandle, tests: Iterable[Test], hooks: Iterable[Hook])
 
         if coverage is not None:
             self.log.info("Enabling coverage collection of Python code")
-            self._cov = coverage.coverage(branch=True, omit=["*cocotb*"])
+            # TODO: We want to exclude cocotb code from coverage collection and
+            # only collect coverage for user code. Using omit as shown below is
+            # a hackish workaround to exclude the installed cocotb package, but
+            # it is nowhere near fool-proof.
+            self._cov = coverage.coverage(branch=True, omit=["*site-packages/cocotb/*"])
```

Comment:
This assumes the user installed cocotb typically. This won't work for my current setup.
```suggestion
            cocotb_package_dir = os.path.dirname(__file__)
            self._cov = coverage.coverage(branch=True, omit=["{}/*".format(cocotb_package_dir)])
```

Reply:
Good idea! Done that. (Forward slashes are OK on Windows here, see https://github.com/cocotb/cocotb/pull/1896#discussion_r433155085)